bindings/python/examples/client/post_raw_block.py

```diff
@@ -11,4 +17,4 @@
 result = client.post_block_raw(blockBytes)
 
 # Print block raw
-print(f'{result}')
+print(f'Posted raw block: {os.environ["EXPLORER_URL"]}/block/{result}')
```

Comment:
This doesn't have a fallback right? Would it crash? Should we have a fallback?

Reply:
yea it would throw an error `KeyError: 'EXPLORER_URL'`. We dont have in rust and node, so i think its fine.